pkg/resource/manager.go

```diff
@@ -91,9 +94,7 @@ func (m *manager) BatchCreateGenericResources(ctx context.Context, request *pb.B
 		log.Debug("Adding resource to bulk request", zap.String("resourceName", resourceName))
 
 		bulkCreateItems = append(bulkCreateItems, &esutil.BulkCreateRequestItem{
-			Message: &pb.GenericResource{
-				Name: resourceName,
-			},
+			Message:    genericResources[resourceName],
 			DocumentId: resourceName,
```

Comment:
Is this a good time to change this to the name + type to prevent collisions?

Reply:
yeah, might as well